aptos-move/framework/aptos-framework/sources/configs/version.move

```diff
@@ -28,13 +28,11 @@ module aptos_framework::version {
     }
 
     /// Updates the major version to a larger version.
-    /// This is only used in test environments and outside of them, the core resources account shouldn't exist.
+    /// This can be called by on chain governance.
     public entry fun set_version(account: &signer, major: u64) acquires Version {
```

Comment:
maybe relabel this to aptos_framework?

Reply:
Just updated this. It can be either aptos framework or core resources (if it's tests/testnets).